katharsis-jpa/src/main/java/io/katharsis/jpa/internal/query/AbstractQueryExecutorImpl.java

```diff
@@ -95,8 +95,7 @@ public MetaDataObject getMeta() {
 				entityList.add((T) values[0]);
```

Comment:
java 8 isn't _new_ - java 9 is due out this summer. This isn't about performance. It's more about reduction of the code base.